Makefile

```diff
@@ -107,11 +104,11 @@ ghcid-cli :
 
 .PHONY: doctest
 doctest :
-	$(DOCTEST) Cabal-syntax
-	$(DOCTEST) Cabal-described
-	$(DOCTEST) --build-depends=QuickCheck Cabal
-	$(DOCTEST) cabal-install-solver
-	$(DOCTEST) cabal-install
+	cd Cabal-syntax && $(DOCTEST)
+	cd Cabal-described && $(DOCTEST)
+	cd Cabal && $(DOCTEST)
+	cd cabal-install-solver && $(DOCTEST)
+	cd cabal-install && $(DOCTEST)
```

Comment:
@geekosaur changing the working directory is necessary as per https://github.com/haskell/cabal/issues/9115#issuecomment-2254293615.

Other than that, this is good to go.